.github/workflows/open-issue-in-repo.yml

```diff
@@ -1,41 +1,17 @@
 # **what?**
 # Open an issue in a specified repository with a specified title and body.  If the issue already exists, do nothing.
-# The calling workflow should be triggered off an issue or PR.
 
 # **why?**
 # To be able to trigger issues to open in other repositories from a workflow in a different repository.
 
 # **when?**
 # This will be triggered from another workflow on an issue or PR.
-# This could also be triggered when a label is added to an issue or PR.
+# The calling workflow determines what trigger to use.
 
 
 name: Open Issue in Another Repository
 
 on:
-  workflow_dispatch:  # for testing
```

Comment:
This no longer works with the addition of the PAT for auth.  It's left over from initial testing.